docker/gcp_pudl_etl.sh

```diff
@@ -71,7 +71,7 @@ function run_pudl_etl() {
 
 function save_outputs_to_gcs() {
     echo "Copying outputs to GCP bucket $PUDL_GCS_OUTPUT" && \
-    gsutil -m cp -r "$PUDL_OUTPUT" "$PUDL_GCS_OUTPUT" && \
+    gsutil -q -m cp -r "$PUDL_OUTPUT" "$PUDL_GCS_OUTPUT" && \
```

Comment:
I added some `-q` and `--quiet` flags to the GCS and S3 commands since they produce verbose interactive terminal outputs that we don't want to log.